src/ucp/api/ucp.h

```diff
@@ -168,6 +172,10 @@ enum ucp_dt_type {
                                                                 mask */
 };
 
+struct ucp_worker_params {
+    cpu_set_t                     cpu_set; /* cpu_set affinity to use when creating a cq */
```

Comment:
* pls use doxygen style comments and document the structure as well
* add field_mask for compatibility, see ucp_context_params_t